<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1868</width>
    <height>1079</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车载全景拼接系统</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,14">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="logo_text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>530</width>
           <height>72</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>530</width>
           <height>72</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>40</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(252, 175, 62);</string>
         </property>
         <property name="text">
          <string>车载全景拼接系统</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="systemSelect">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="bowl">
        <attribute name="title">
         <string>效果展示</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
             <item>
              <widget class="QOpenGLWidget" name="renderView"/>
             </item>
             <item>
              <widget class="QLabel" name="stitchView">
               <property name="styleSheet">
                <string notr="true">QLabel{background-color:rgb(0, 0, 0);}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="lineWidth">
                <number>2</number>
               </property>
               <property name="text">
                <string>全景拼接图</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
             <item>
              <widget class="QTabWidget" name="inputSelect">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="image">
                <attribute name="title">
                 <string>输入参数</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="0">
                    <widget class="QLabel" name="dataPathLabel">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>数据目录: </string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="dataPath">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QToolButton" name="dataPathBtn">
                     <property name="text">
                      <string>...</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_8">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>参数文件: </string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="paraPath">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QToolButton" name="paraBtn">
                     <property name="text">
                      <string>...</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_9">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>演示总帧数: </string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="maxFrame">
                     <property name="readOnly">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_6">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>彩色扩展名: </string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QComboBox" name="colorExt">
                     <item>
                      <property name="text">
                       <string>.jpg</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>.png</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="label_7">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>深度扩展名: </string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="1">
                    <widget class="QComboBox" name="depthExt">
                     <item>
                      <property name="text">
                       <string>.png</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>.jpg</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="5" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>相机数量: </string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="5" column="1">
                    <widget class="QComboBox" name="camNum">
                     <item>
                      <property name="text">
                       <string>4</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="6" column="0" colspan="2">
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>15</width>
                         <height>17</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <widget class="QPushButton" name="startBtn">
                         <property name="text">
                          <string>启动</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="stopBtn">
                         <property name="text">
                          <string>停止</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_3">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>14</width>
                         <height>17</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="cliResult">
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOn</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOn</enum>
               </property>
               <property name="lineWrapMode">
                <enum>QTextEdit::NoWrap</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="dataTypeGroup"/>
  <buttongroup name="renderPatternGroup"/>
 </buttongroups>
</ui>
